CHANGES/7942.feature

```diff
@@ -0,0 +1 @@
+Improve sync performance with no-op when possible.
```

Comment:
Please add the fact that the no-op can be disabled with the `optimize=False` sync parameter also.

Reply:
it is disabled by default, we need to set it to true to enable

Reply:
Oh I think optimize should be enabled by default.

Regardless though, we need to identify that this new option is available to the user.

Reply:
I think we definitly want it enabled by default. This is also how other content types work, e.g. rpm.